server/api/errors.go

```diff
@@ -22,29 +22,30 @@ type ErrorSource struct {
 	Parameter string `json:"parameter,omitempty"`
 }
 
-func RenderError(w http.ResponseWriter, statuscode int, err APIError) {
+// TODO: rename to Render
```

Comment:
If this was a method on the `Error`, then you'd be able to rename it as `Render` without losing the context of it being about an error.

Reply:
I like this plan 👍